compass_vue/pages/home.vue

```diff
@@ -8,165 +8,217 @@
+          <div class="input-group input-group-sm mb-0">
+            <input
+              type="text"
+              class="form-control"
+              :placeholder="'Student ' + searchOption.label + ' Contains...'"
+              :aria-label="'Student ' + searchOption.label + ' Contains...'"
+              v-model="searchText"
+              aria-describedby="button-addon2"
+            />
+            <button
+              class="btn btn-outline-primary"
+              type="button"
+              id="button-addon2"
+              v-on:click="loadStudentList"
+            >
+              GO
+            </button>
           </div>
         </div>
-        <div class="col-sm-4">
-          <div class="small">Select advisor caseload:</div>
+        <div class="col-sm-2">
+          <div class="small lh-lg">Select advisor caseload:</div>
           <select class="form-select form-select-sm" aria-label="Default select example">
             <option selected>All advisers</option>
             <option value="1">Jon Average</option>
             <option value="2">April Foolery</option>
             <option value="3">Bob Samsonite</option>
           </select>
         </div>
-      </div>
-      <div class="row">
-        <div class="col">
-          <table class="table table-striped table-hover border">
-            <thead>
-              <tr>
-                <th scope="col"></th>
-                <th scope="col">Student Name</th>
-                <th scope="col">Student Number</th>
-                <th scope="col">UW NetID</th>
-                <th scope="col">Class</th>
-                <th scope="col">Major</th>
-                <th scope="col">Enroll Status</th>
-                <th scope="col">Gender</th>
-                <th scope="col">Adviser</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr v-for="(item,) in students" :key="item.SystemKey">
-                <td>
-                  <div style="width:40px;">
-                    <img src="/static/img/napoleon-dynamite.jpeg" class="img-fluid rounded-circle border border-3" alt="">
-                  </div>
-                </td>
-                <td>{{item.student_name}}</td>
-                <td>
-                  <router-link :to="{ name: 'Student', params: { id: item.student_number } }">
-                    {{item.student_number}}
-                  </router-link>
-                </td>
-                <td>{{item.uw_net_id}}</td>
-                <td>{{item.class_desc}}</td>
-                <td>{{item.major_full_name}}</td>
-                <td>{{item.enrollment_status_desc}}</td>
-                <td>{{item.gender}}</td>
-                <td>{{item.adviser_full_name}}</td>
-              </tr>
-            </tbody>
-          </table>
+        <div class="col-sm-6">
+          <div class="small lh-lg text-end">Displaying: {{ studentsCount }} students</div>
```

Comment:
This always says displaying 100 students. It should instead say the something along the lines of "Displaying 50-100 of 100 students". It's fixed to the total students count which doesn't seem correct.

Reply:
this was originally in the pagination template. but now that display the pagination at the top and bottom, i didn't want to necessarily display the message in both locations. i'll make a prop that controls show/hide of the message

Reply:
The problem is that the message being shown seems incorrect. It says "Displaying: 100" when there are only 50 per page.